_projects/meta_review.md

```diff
@@ -0,0 +1,57 @@
+---
+collaborating_projects:
+  - coala
+desc: "Implement a meta-review system that tracks and provides meaningful analysis of reactions on reviews."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - "https://github.com/coala/projects/issues/480"
+markdown: meta_review.md
+mentors:
+  - hemangsk
+  - blazeu
+  - alisianoi
+  - andrewda
+name: "Meta-review"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "The applicant should be familiar with GitHub GraphQL API and javascript."
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The goal of project meta-review is to build a meta-review track system similar to a meta-moderation system that can track emoji responses to review comments. These responses are to be collected, processed and displayed on [gh-board][https://github.com/coala/gh-board], which is a nice serverless kanban board.
```

Comment:
add comma before and after "similar to a meta-moderation system"

Reply:
Break lines at 80cpl

Reply:
Note that this project is also about encouraging people to do meta-review of PR comments, systematically.

Currently, we do not do this systematically, or even occasionally.  emoji use is currently hap-hazard, and probably the currently implied meaning of `:+1:` is "OK, I will do this", not "I agree"+"I found your communication style appropriate".

This actually presents a problem for gh-board, as its UI is currently centric to the issue/PR, and it does not display issue comments. The granularity of gh-board is wrong.  How then do we use gh-board to show which PR reviews have not been meta-reviewed yet.  Worst case is that the tracking stores in the json which PR do not have meta-reviews, and gh-board shows only a list of those PRs (and the user must then find the reviews in GitHub PR webapp, and do meta-review there).  An improvement is to store the identifier of the PR comments which do not have meta-reviews, and gh-board links to the PR review comment in GitHub PR webapp.  In the ultimate version, gh-board would show the review comments.

Reply:
It is important to note in this project that the scope will exclude non-public meta-reviews.

That was a goal of the original design, but it introduces too many additional complexities into the first iteration.  We need to try using the public only version for long time before adding support for non-public meta-reviews.